File: Android/sources/jp/co/cyberagent/android/gpuimage/GPUImageNativeLibrary.java
package jp.co.cyberagent.android.gpuimage;

import android.graphics.Bitmap;

public abstract class GPUImageNativeLibrary {
    static {
        System.loadLibrary("yuv-decoder");
    }

    public static native void YUVtoRBGA(byte[] bArr, int i, int i2, int[] iArr);

    public static native void adjustBitmap(Bitmap bitmap);
}

File: IOS/v2/jp/co/cyberagent/android/gpuimage/GPUImageNativeLibrary.swift
package jp.co.cyberagent.iOS.gpuimage;.swift
import iOS.graphics.Bitmap;.swift
public abstract class GPUImageNativeLibrary {.swift
static {.swift
System.loadLibrary("yuv-decoder");.swift
}.swift
public static native void YUVtoRBGA(byte[] bArr, int i, int i2, int[] iArr);.swift
public static native void adjustBitmap(Bitmap bitmap);.swift
}.swift
